articles/how-kubernetes-pods-work/index.md

```diff
@@ -0,0 +1,65 @@
+In [cloud computing](https://www.section.io/engineering-education/introduction-to-cloud-computing/), a pod is a high-level structure and the tiniest execution object in [Kubernetes](https://www.section.io/engineering-education/what-is-kubernetes/). These pods are temporary such that if one fails, Kubernetes simultaneously models a copy such that the work keeps going on. They are usually in one or several collections.
```

Comment:
>  These pods are temporary such that if one fails, Kubernetes simultaneously models a copy such that the work keeps going on. They are usually in one or several collections.

Could you please break this down into smaller sentences? It would be easier to read